client/js/lounge.js

```diff
@@ -331,7 +331,7 @@ $(function() {
 		const state = {};
 
 		if (self.hasClass("chan")) {
-			state.clickTarget = `.chan[data-id="${self.data("id")}"]`;
+			state.clickTarget = `#sidebar .chan[data-id="${self.data("id")}"]`;
```

Comment:
This is just a safeguard as I realize that `.chan[data-id="${self.data("id")}"]` actually refers to 2 things, the sidebar element and the chat window element. Triggering a click on the chat window element does nothing so we're safe for now, but if this changes someday we'd have some weird behavior to debug.
Also there is no reason to click on it so might as well be selective.